src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/CaseEndpoint.java

```diff
@@ -31,6 +38,60 @@ public CaseEndpoint(UserIdentity userIdentity, CaseService caseService) {
     this.caseService = caseService;
   }
 
+  @PostMapping("{caseId}/action/updateSensitiveField")
+  public ResponseEntity<?> updateSensitiveField(
+      @PathVariable(value = "caseId") UUID caseId,
+      @RequestBody UpdateSampleSensitive updateSampleSensitive,
+      @Value("#{request.getAttribute('userEmail')}") String userEmail) {
+
+    Case caze = caseService.getCaseByCaseId(caseId);
+
+    userIdentity.checkUserPermission(
+        userEmail,
+        caze.getCollectionExercise().getSurvey(),
+        UserGroupAuthorisedActivityType.UPDATE_SENSITIVE_SAMPLE);
+
+    Optional<List<String>> validationErrorsOpt =
```

Comment:
I don't think the optional is adding any benefit here. Empty list = zero errors = OK request, IMHO